deploy/monitor/mdc_prometheus_rule.yaml

```diff
@@ -8,13 +8,41 @@ metadata:
   name: mdc-monitoring
 spec:
   groups:
-    - name: general.rules
-      rules:
-      - alert: MobileDeveloperConsoleDown
-        expr: absent(up{job="example-mdc-mdc"} == 1)
-        for: 5m
-        labels:
-          severity: critical
-        annotations:
-          description: "The Mobile Developer Console has been down for more than 5 minutes."
-          summary: "The Mobile Developer Console is down."
+  - name: general.rules
+    rules:
+    - alert: MobileDeveloperConsoleContainerDown
+      expr: absent(kube_pod_container_status_running{namespace="mobile-developer-console",container="mdc"}>=1)
+      for: 5m
+      labels:
+        severity: critical
+      annotations:
+        description: "The MDC has been down for more than 5 minutes. "
+        summary: "The mobile-developer-console is down. For more information see on the MDC at https://github.com/aerogear/mobile-developer-console"
+        sop_url: "https://github.com/aerogear/mobile-developer-console-operator/blob/0.1.0/SOP/SOP-mdc.adoc"
```

Comment:
Let's update this one now to 0.1.1 now as per the conversation b y @aliok and @grdryn in chat.

Reply:
@laurafitzgerald as described in the README it should be updated when a release is made.